src/db/sqldb/sqldb_suite_test.go

```diff
@@ -116,5 +116,21 @@ func insertAppMetric(appId string) {
 	if e != nil {
 		Fail("can not insert data to policy table: " + e.Error())
 	}
+}
+
+func cleanScalingHistoryTable() {
+	_, e := dbHelper.Exec("DELETE from scalinghistory")
```

Comment:
this will need to expose the db connection to public.  And I want to separate the one used for preparing the test and the one that is under test.